openmc/mgxs/mgxs.py

```diff
@@ -212,11 +229,70 @@ def num_groups(self):
         return self.energy_groups.num_groups
 
     @property
+    def scores(self):
+        return ['flux', self.rxn_type]
+
+    @property
+    def filters(self):
+        group_edges = self.energy_groups.group_edges
+        energy_filter = openmc.Filter('energy', group_edges)
```

Comment:
Oh sorry, that was confusing. What I meant is that `[id(f) for f in mgxs_object.filters]` will return a different list each time you evaluate it.